volume-providers/vpc/provider/util.go

```diff
@@ -28,6 +28,11 @@ var maxRetryGap = 60
 // retryGap ...
 var retryGap = 10
 
+//ConstantRetryGap ...
+const (
+	ConstantRetryGap = 10 // seconds
```

Comment:
Are we not reading RetryGap from configurations ?

Reply:
no this we discussed also along with Narasimha, conclusion was 'lets not have too many variable expose to user'